src/sentry/models/grouptombstone.py

```diff
@@ -39,6 +39,8 @@ class GroupTombstone(Model):
         blank=True, null=True
     )
     actor_id = BoundedPositiveIntegerField(null=True)
+    times_seen = BoundedPositiveIntegerField(default=0, null=True)
+    last_seen = models.DateTimeField(default=timezone.now, null=True)
```

Comment:
Do you actually want these to be null by default? You can use `db_default` if you want to give existing rows a value

Reply:
oh no. that is not what I want. I would like existing rows to not have any value. Hopfully the buffer_inc func deals with null values